### ISSUE TITLE
API "_node/stats/pipelines" always show one pipeline stat

### ISSUE THREAD

**Issue by archichen** [elastic/logstash#11210]
- Version: logstash 7.3.1
- Operating System: Windows 10 1903
- Config File (if you have sensitive info, please remove it):
logstash.yml
```YAML
node.name: win-logstash
config.reload.automatic: true
config.reload.interval: 3s
```
pipelines.yml
```YAML
- pipeline.id: test1
  path.config: D:\data\logstash\t1.conf
  pipeline.workers: 1
- pipeline.id: test2
  path.config: D:\data\logstash\t2.conf
  pipeline.workers: 1
```
- Steps to Reproduce:
Start logstash `logstash -r`

Monitor API only shows one pipeline stat but I have tow.
![image](https://user-images.githubusercontent.com/18643080/66643622-b2eea080-ec51-11e9-8bb7-0429473f5f74.png)

**Comment by TheVastyDeep** [comment]
That would appear to duplicate [this](https://github.com/elastic/logstash/issues/11021). Fixed in 7.4

**Comment by kares** [comment]
definitely fixed by now
